UtilityApps/CMakeLists.txt

```diff
@@ -42,10 +42,13 @@ if(TARGET Geant4::Interface)
 endif()
 
 # the web display tool depends on the ROOT7 version of DDEve
-if(TARGET ROOT::ROOTEve)
+find_package (ROOT COMPONENTS ROOTEve)
+if(ROOTEve_FOUND)
```

Comment:
I don't see ROOTEve_FOUND being defined in the ROOTConfig.cmake.
Just rely on the existence of the TARGET please!

Reply:
This is my bad - the usual CMake style should have defined ROOTEve_FOUND after a find_package(ROOTEve) call, but looks like it's not working here.